CmdStarter.Tests/CommandsBasics.cs

```diff
@@ -294,46 +294,52 @@ public void EnsuresNoWrongClassesTypes()
         [TestCase<ArgParent>]
         [TestCase<ArgChild>]
         [TestCase<FirstByInterface>]
+        [TestCase<SecondByInterface>]
         public void FindsCommand<CommandType>() where CommandType : IStarterCommand
         {
             starter.InstantiateCommands();
             Assert.That(starter.FindCommand<CommandType>(), Is.Not.Null);
         }
 
-        [TestCase(nameof(OptHandling.MyOptInt), OptHandling.MY_OPT_INT_KEBAB, 111, 999)]
-        [TestCase(nameof(OptHandling.MyOptBool), OptHandling.MY_OPT_BOOL_KEBAB, false, true)]
-        [TestCase(nameof(OptHandling.MyOptListInt), OptHandling.MY_OPT_LIST_INT_KEBAB, null, new int[] {11, 22})] 
-        public async Task IsPropertyFilledWithOption(string propertyName, string optionName, object defaultValue, object expectedValue)
+        [TestCase<OptHandling>(nameof(OptHandling.MyOptInt), OptHandling.MY_OPT_INT_KEBAB, 111, 999)]
+        [TestCase<OptHandling>(nameof(OptHandling.MyOptBool), OptHandling.MY_OPT_BOOL_KEBAB, false, true)]
+        [TestCase<OptHandling>(nameof(OptHandling.MyOptListInt), OptHandling.MY_OPT_LIST_INT_KEBAB, null!, new int[] {11, 22})]
+        [TestCase<OptByInterface>(nameof(OptByInterface.IntOpt), OptByInterface.INT_OPT_KEBAB, 111, 999)]
+        public async Task IsPropertyFilledWithOption<CommandType>(string propertyName, string optionName, object defaultValue, object expectedValue) where CommandType : class, IStarterCommand
         {
-            var propertyTested = typeof(OptHandling).GetProperty(propertyName);
+            var propertyTested = typeof(CommandType).GetProperty(propertyName);
 
             // Test without option for default value
-            starter.Namespaces = starter.Namespaces.Add(typeof(OptHandling).Namespace!);
-            await starter.Start(new string[] { nameof(OptHandling).PascalToKebabCase() });
-            var optionCommand = starter.FindCommand<OptHandling>() as OptHandling;
+            var commandType = typeof(CommandType);
```

Comment:
Could be defined above and use the variable `commandType` instead of `typeof` in first line.